new/track-rfcs.md

```diff
@@ -0,0 +1,96 @@
+---
+RFC: unassigned
+Author: John Keiser <jkeiser@chef.io>
+Status: Draft
+Type: Process
+---
+
+# Tracking for Standards Track RFCs
+
+Standards Track RFCs are intended for implementation.  This proposal aims to
+make it easier to tell when an RFC is implemented or orphaned, or what its
+implementation status is.
+
+## Motivation
+
+    As a Chef Contributor,
+    I want to know whether an RFC is actually being worked on,
+    so that I can decide whether to wait, write it myself, or write a replacement RFC.
+
+    As a Chef Contributor,
+    I want to know whether an RFC has become orphaned,
+    So that I can decide whether to find a new owner for the subject.
+
+## Specification
+
+### Tracking Metadata Field
+
+RFCs will now include a metadata field called Tracking, which will generally be
+a list of URLs to issues or PRs where the feature is tracked, and may also link
+to blog posts about relevant releases.
+
+For example, rfc015 metadata would look like this:
+
+```yaml
+---
+RFC: 34
+Author: Daniel DeLeo <dan@getchef.com>
+Status: Final
+Type: Standards Track
+Chef-Version: 12.0
+Tracking:
+  - https://www.chef.io/blog/2014/11/25/ruby-1-9-3-eol-and-chef-12/
+---
+```
+
+And the rfc024 metadata would look like this:
+
+```yaml
+---
+RFC: 24
+Author: John Keiser <jkeiser@getchef.com>
+Status: Accepted
+Type: Standards Track
+Chef-Version: 12
+Tracking:
+  - https://github.com/chef/chef/pull/1969
+---
+```
+
+ALL accepted Standards Track RFCs will have:
+
+- Status of Accepted or Final
+- Tracking URLs (unless it is an old RFC with Final status, in which case
+  Tracking URLs are optional)
+- Chef-Version (if Final)
+
+### RFC "Orphaned" status
```

Comment:
As discussed in today's IRC meeting, we should use the existing Withdrawn status from [rfc000](https://github.com/chef/chef-rfc/blob/master/rfc000-rfc-process.md#rfc-status-reference).